Library/test-helpers/lib.sh

```diff
@@ -2217,6 +2249,50 @@ limeconRunSystemd() {
 true <<'=cut'
 =pod
 
+=head2 limeconRunTenant
+
+Tenant container run via podman with specified parameters.
+
+    limeconRunTenant NAME TAG IP NETWORK
+
+=item NAME
+
+Set name of container.
+
+=item TAG
+
+Name of image tag.
+
+=item IP
+
+IP address of container.
+
+=item NETWORK
+
+Name of used podman network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconRunTenant() {
+
+    local NAME=$1
+    local TAG=$2
+    local IP=$3
+    local NETWORK=$4
+
+    limeconRun $NAME $TAG $IP $NETWORK "/sbin/init" "--volume=/etc/keylime/:/etc/keylime/"
```

Comment:
In my opinion this function should simply start the container, running the keylime_tenant command and passing the required arguments.. and finish after that. I.e. there won't be no cert copying, those would be mounted to the container when starting, and there won't be `podman exec`.

Reply:
Should be done for now.